sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStore.scala

```diff
@@ -67,6 +67,15 @@ trait ReadStateStore {
   def get(key: UnsafeRow,
     colFamilyName: String = StateStore.DEFAULT_COL_FAMILY_NAME): UnsafeRow
 
+  /**
+   * Provides an iterator containing all values of a non-null key.
```

Comment:
Let's make a contract about expected return value, especially about clarification of value for non-existing key. Multiple implementations should follow the contract so this needs to be clearly explained.

Reply:
Sounds good. Done